clients/drcachesim/simulator/prefetcher.h

```diff
@@ -51,7 +51,7 @@ class prefetcher_t {
     {
     }
     virtual void
-    prefetch(caching_device_t *cache, const memref_t &memref);
```

Comment:
I know it's fairly obvious, but I suggest a comment to explicitly state that prefetch() is called for all accesses, and "missed" is how you know if this memref.data.addr is already in the cache or not.

Reply:
Done.